pkg/kritis/review/review.go

```diff
@@ -52,12 +52,39 @@ func New(client metadata.Fetcher, c *Config) Reviewer {
 	}
 }
 
+func imageInGapAllowlist(image string, gap v1beta1.GenericAttestationPolicy) bool {
+	for _, i := range gap.Spec.ImageAllowlist {
+		if i == image {
```

Comment:
Let's make it match the semantics of the whitelist in Binary Authorization:
https://cloud.google.com/binary-authorization/docs/policy-yaml-reference#admissionwhitelistpatterns